src/amo/reducers/reviews.js

```diff
@@ -227,12 +229,32 @@ export const selectLatestUserReview = ({
   return selectReview(reviewsState, userReviewId);
 };
 
+// This is used in the SET_REVIEW and SET_INTERNAL_REVIEW cases.
```

Comment:
Do you think this comment is necessary? Usually when I want to see where a function is used I just search for it. Comments like tend to go stale quickly, too.